src/mds/Migrator.cc

```diff
@@ -1665,7 +1654,7 @@ uint64_t Migrator::encode_export_dir(bufferlist& exportbl,
 
 void Migrator::finish_export_dir(CDir *dir, mds_rank_t peer,
 				 map<inodeno_t,map<client_t,Capability::Import> >& peer_imported,
-				 list<MDSInternalContextBase*>& finished, int *num_dentries)
+				 list<MDSInternalContextBase*>& finished, unsigned& num_dentries)
```

Comment:
This isn't relevant to this PR. Needs to be a separate refactor PR that won't be backported.